cfme/tests/infrastructure/test_vm_migrate.py

```diff
@@ -39,7 +40,8 @@ def test_vm_migrate(appliance, new_vm, provider):
         test_flag: migrate, provision
     """
     # auto_test_services should exist to test migrate VM
-    vm_host = new_vm.get_detail(properties=('Relationships', 'Host'))
+    view = navigate_to(new_vm, 'Details')
+    vm_host = view.entities.summary('Relationships').get_text_of('Host')
     migrate_to = [vds.name for vds in provider.hosts if vds.name not in vm_host][0]
```

Comment:
This is hitting an IndexError in PRT, but I don't think your change is impacting it.

Reply:
Correct, I can see the same error in Jenkins.

Reply:
Actually it's fixed already in #6664